src/Microsoft.ML.AutoMLFeaturizers/Microsoft.ML.AutoMLFeaturizers.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeInPackage>Microsoft.ML.Featurizers</IncludeInPackage>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DefineConstants>CORECLR</DefineConstants>
```

Comment:
I don't think this `DefineConstant` is being used in the new assembly. It can be removed.

Reply:
Same comment as the prior one. This was removed last night, but is not showing up in the PR for some reason even though it is in my branch. Will rebase and check again.